### ISSUE TITLE
Incomes and Payments filtering

### ISSUE THREAD

**Issue by prekucki** [golemfactory/golem-client#14]
Incomes and Payments should have additional arguments printed out at first, as it is not obvious that user can filter them with additional commands

#### Incomes
* All
* Awaiting
* Confirmed
* Total (or just print sum under each above)

#### Payments
As above

**Comment by prekucki** [comment]
```
$ golemcli incomes -h
golemcli-incomes 0.1.0
Golem Factory <contact@golem.network>
Display incomes

USAGE:
    golemcli incomes [FLAGS] [OPTIONS] [filter_by]

FLAGS:
        --json    Return results in JSON format
        --full    Show full table contents
    -h, --help    Prints help information

OPTIONS:
    -a, --address <rpc_address>    Golem node's RPC address
    -p, --port <rpc_port>          Golem node's RPC port
    -d, --datadir <data_dir>       Golem node's data dir
        --sort <sort_by>           Sort incomes [possible values: payer, status, value]

ARGS:
    <filter_by>    Filter by status [possible values: awaiting, confirmed]
```

as I understand we want to make filter_by as a subcommand,

**Comment by kmazurek** [comment]
The `payments` and `incomes` sections should follow the convention of using a `show` subcommand in order to display their respective data.

**Comment by jamuszyn** [comment]
Would work if `payments` would print:

USAGE:
    payments [FLAGS] <SUBCOMMAND>

FLAGS:
        --json    Return results in JSON format
    -h, --help    Prints help information

SUBCOMMANDS:
    help    Prints this message or the help of the given subcommand(s)
    show    Show payments

OPTIONS:
        --sort <sort_by>    Sort payments [possible values: subtask, payee, status, value, fee]

ARGS:
    <filter_by>    Filter by status [possible values: awaiting, confirmed]

---

Additionally **Incomes** as well as **Payments** table should have summary printed out below the table. 
Summary `status` `Total Value` for each status available (confirmed, awaiting, overdue, etc.)

**Comment by prekucki** [comment]
resolved in 1.9
